frontend/src/lib/utils/actions.ts

```
@@ -86,7 +86,11 @@ export async function handleErrorResponse({
 		return message(form, { error: res.error || res.detail });
 	}
 	Object.entries(res).forEach(([key, value]) => {
-		setError(form, key, safeTranslate(value));
+		if (Array.isArray(value)) {
+			value.forEach((item: string) => setError(form, key, safeTranslate(item)));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)